src/main/java/zeronote/userinterface/command/timetable/DoneCommandTimetableMode.java

```diff
@@ -10,6 +10,11 @@
 import zeronote.userinterface.CliMessages;
 import zeronote.userinterface.command.CliCommand;
 
+//@@author yAOwzers
+/**
+ * Marks a specific task in the tasklist in Zer0Note as done.
+ *
```

Comment:
I thought javadoc doesn't have this extra new line at the end, but it passed the test so i guess okay hehe.